github/dependabot_secrets.go

```diff
@@ -162,7 +163,7 @@ func (s *DependabotService) CreateOrUpdateRepoSecret(ctx context.Context, owner,
 func (s *DependabotService) CreateOrUpdateOrgSecret(ctx context.Context, org string, eSecret *DependabotEncryptedSecret) (*Response, error) {
 	repoIDs := make([]string, len(eSecret.SelectedRepositoryIDs))
 	for i, secret := range eSecret.SelectedRepositoryIDs {
-		repoIDs[i] = fmt.Sprintf("%v", secret)
+		repoIDs[i] = strconv.FormatInt(secret, 10)
```

Comment:
Yuck.  I do not like this change, as `fmt.Sprintf` is much more common and idiomatic Go.

Reply:
Reverted.